crates/lnd-bridge/examples/mock.rs

```diff
@@ -169,15 +183,18 @@ async fn cancel_invoice(
     }
 }
 
-async fn pay_invoice(Extension(tx): Extension<broadcast::Sender<String>>) -> impl IntoResponse {
+async fn pay_invoice(
+    Extension(tx): Extension<broadcast::Sender<String>>,
+    State(state): State<Arc<RwLock<String>>>,
+) -> impl IntoResponse {
     let message = serde_json::to_string(&InvoiceResult {
         result: Invoice {
             memo: "".to_string(),
             expiry: 0,
             amt_paid_sat: 0,
             state: InvoiceState::Accepted,
             payment_request: "".to_string(),
-            r_hash: "".to_string(),
+            r_hash: state.read().expect("").to_string(),
```

Comment:
🙃 Just allow unwrap in this crate or in this file if you don't want to deal with `expect`.